src/applications/personalization/preferences/containers/PreferencesWidget.jsx

```diff
@@ -38,48 +16,55 @@ class PreferencesWidget extends React.Component {
     this.state = {};
   }
 
-  handleAccordionToggle = e => {
-    e.preventDefault();
+  handleViewToggle = slug => {
     this.setState({
-      [e.target.name]: !this.state[e.target.name],
+      [slug]: !this.state[slug],
     });
   };
 
+  handleRemove = async slug => {
+    await this.props.setPreference(slug, false);
```

Comment:
Good question, how about we create a remove action instead so that the logic is not handled in the component?

Reply:
sooo... I don't really have a strong preference. I don't mind handling the logic here. And I guess since `setPreference` is just a redux action creator we probably don't need to worry about error handling like we would with something making an API call.